demo/src/api-config.js

```diff
@@ -9,8 +9,8 @@ let apiRoot;
 
 const origin = window && window.location && window.location.origin;
 
-if (origin === 'http://localhost:3000') {
-    apiRoot = 'http://localhost:8000';
+if (origin.includes(':3000')) {
```

Comment:
This seems like a good change--but just curious: why was it necessary?

Reply:
Because I was running the demo on an EC2 instance and hitting it from my mac.  I wasn't going to `localhost`,  I was going to `10.12.whatever`.